site/pages/api/duplicate-consequence.api.ts

```diff
@@ -76,25 +70,28 @@ const duplicateConsequence = async (req: NextApiRequest, res: NextApiResponse):
         redirectToWithQueryParams(
             req,
             res,
-            template === "true" ? ["template"] : [],
+            [],
             `${
                 disruption.publishStatus === PublishStatus.draft
                     ? REVIEW_DISRUPTION_PAGE_PATH
                     : DISRUPTION_DETAIL_PAGE_PATH
             }/${validatedBody.data.disruptionId}`,
+            isFromTemplate ? ["isFromTemplate=true"] : [],
         );
 
         return;
     } catch (e) {
         if (e instanceof TooManyConsequencesError) {
```

Comment:
Update error handling here in a similar way to my comment on duplicate-template-consequence.api. Will have to declare publishStatus outside scope of try catch block